src/script/luna/mememu.cpp

```diff
@@ -1305,18 +1308,18 @@ class NPCMemory final : public NpcParent
         insert(0x00000024, &NPC_t::Multiplier);
         insert(0x00000026, &NPC_t::TailCD);
         insert(0x00000028, &NPC_t::Shadow);
-        insert(0x0000002c, &NPC_t::TriggerActivate);
-        insert(0x00000030, &NPC_t::TriggerDeath);
-        insert(0x00000034, &NPC_t::TriggerTalk);
-        insert(0x00000038, &NPC_t::TriggerLast);
-        insert(0x0000003c, &NPC_t::Layer);
+        // insert(0x0000002c, &NPC_t::TriggerActivate);
+        // insert(0x00000030, &NPC_t::TriggerDeath);
+        // insert(0x00000034, &NPC_t::TriggerTalk);
+        // insert(0x00000038, &NPC_t::TriggerLast);
+        // insert(0x0000003c, &NPC_t::Layer);
```

Comment:
By fact, these string fields were never modified by autocode, and probably not going to be modded as Autocode was never able to memhack strings. The strings hacking was added at LunaLua later. So, right. For the worst case, you could use the lambda fields instead to add/remove/get string entry later.

Reply:
Cool! So, when (if) we implement LunaLua, that's the time we will need to add these lambda fields.